source/pdk/1.0/pdk_install.md

```diff
@@ -68,6 +68,10 @@ Download and install the PDK package for Windows systems.
 
 With PowerShell 3.0 or later, PDK loads automatically and the `pdk` command is now available to the prompt. If you are using PDK in PowerShell 2.0, you'll need to load the module and set the correct permissions for PDK.
 
+In some Windows installations trying to run the `pdk` command will complain about execution policy restrictions. In that case set your script execution policy to at least [RemoteSigned](https://msdn.microsoft.com/en-us/powershell/reference/5.1/microsoft.powershell.security/set-executionpolicy) by running `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned`. Alternatively, you can change the `Scope` parameter of the `ExecutionPolicy` to the current session only by running `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`
```

Comment:
That is incorrect. The default ExecutionPolicy depends on windows edition, and local settings. Besides many enterprises, and developers changing the default, for example Home versions of Windows usually have been locked down.

The error message from powershell when people run into this is already quite helpful. Having this here only helps by having the command, background, etc ready for copy&paste, when they need it.